### DISCUSSION TITLE
[Enhancement] Add shortcut support (Keyboard Shortcuts)

### DISCUSSION THREAD

**Discussion by Sillyvan** [ciderapp/Cider#676] [Feature Request]
I was thinking of simple player shortcuts for pausing, next song etc..

**Comment by coredev-uk** [comment]
Expand on what your asking, base functionality for Windows and Linux media players has been added. People with media keys can pause/play if they want. If you want custom keyboard shortcuts for pause/playing install an application for this.

**Comment by Sillyvan** [comment]
i was mostly rereferring to what spotify does.
https://support.spotify.com/us/article/keyboard-shortcuts/

im not saying what we need all of those. but for me and surely a lot of people its very annoying that pressing "space" wont play/pause music

**Comment by coredev-uk** [comment]
Ok, thanks for clarifying.

**Comment by coredev-uk** [comment]
Additional request from gms10ur in Discord:
> gms10ur — 195634267403976704
keyboard support;
keyboard navigation and some basic shortcuts;
like press enter to play, del to delete, multiple selection with ctrl and shift
number keys for playlist select etc

**Comment by coredev-uk** [comment]
Expanded implementation for Linux: #105 (Add Vim like bindings for controlling this app):

> It'd be great if this app can be controlled just by keyboard. I'm using cider on arch and every app that I'm using has vim bindings for navigation. Using keyboard for navigation is more time saving and productive as compared to hopping between mouse and KB.
> 
> A solution would be to incorporate qutebrowser like "hint" suggestions, where all the hyperlinks are highlighted with various combinations of letters and the user just has to press the keys which corresponds to the link he wants to go.

**Comment by quacksire** [comment]
No VIM bindings (yet!) but there are shortcuts in the dev branch
